README.md

```diff
@@ -6,6 +6,28 @@ This project aims to make journalists' experience working with SecureDrop less o
 
 This project is under active development, has known bugs and shortcomings, and is not ready for end users. This README is geared towards interested technical users and developers.
 
+### Detailed rationale
+
+SecureDrop's [architecture](https://docs.securedrop.org/en/latest/overview.html#infrastructure) and [threat model](https://docs.securedrop.org/en/stable/threat_model/threat_model.html) are proven, but the current approach also has major drawbacks:
+
+- Journalists must access a separate, airgapped device to even validate that a submission is relevant. This is very onerous, and may reduce the reliance on SecureDrop overall.
+
+- The complexity of the setup and the usage procedures create operational security risks. For example, journalists may accidentally boot up the wrong device using the SVS USB drive, breaking the airgap.
```

Comment:
We can add: the cumbersome submission workflow that must be followed even for spam submissions may cause journalists to develop workarounds that reduce the overall security of the SecureDrop platform.